src/main/jhipster/src/main/webapp/app/study-manager/advance/advance-samples.component.html

```diff
@@ -268,4 +251,112 @@ <h4 jhiTranslate="advance-study.reps.title"></h4>
 		</button>
 	</div>
 </div>
+<div *ngIf="isAttributesPropagationView">
```

Comment:
It would be nice if you extract this view into a separate component, to avoid code duplication.